CCIT_Portal/templates/account/login.html

```diff
@@ -1,44 +1,113 @@
-{% extends "account/base.html" %}
-
+<!DOCTYPE html>
+{% load static %}
 {% load i18n %}
 {% load account socialaccount %}
```

Comment:
Remove all the tags with i18 or trans and put there the actual things you want to display